dds/idl/marshal_generator.cpp

```diff
@@ -843,33 +1012,78 @@ namespace {
       extraction.addArg("strm", "Serializer&");
       extraction.addArg(sf.arg_.c_str(), sf.ref_);
       extraction.endArgs();
+
+      be_global->impl_ <<
+        "  const Encoding& encoding = strm.encoding();\n"
+        "  ACE_UNUSED_ARG(encoding);\n";
+      std::vector<string> code;
```

Comment:
Is this collection of source code lines really needed?  Could it just be a singe string with embedded newlines?